readme.md

```diff
@@ -32,3 +32,12 @@ Missing something awesome? Anyone is welcome to [submit projects to this list](h
 * [ModelOriented/MAIR](https://github.com/ModelOriented/MAIR): Monitoring impact of AI regulations with a DVC pipeline.
 * [Kaggle-Titanic-DVC](https://dagshub.com/kingabzpro/kaggle-titanic-dvc): Survival analysis DVC experiment.
 * [VQA-With-Multimodal-Transformers](https://github.com/tezansahu/VQA-With-Multimodal-Transformers): Visual Question Answering task on the [DAQUAR Dataset](https://www.kaggle.com/tezansahu/processed-daquar-dataset) using multimodal [transformer models](https://huggingface.co/docs/transformers/index) with an experiment pipeline tracked in DVC Studio.
+
+## Iterative Tools in Research Papers 
+* [On the Co-evolution of ML Pipelines and Source Code](https://mcis.cs.queensu.ca/publications/2021/saner.pdf)  - Empirical Study of DVC Projects.
+* [FAIRly big](https://www.nature.com/articles/s41597-022-01163-2) -  A framework for computationally reproducible processing of large-scale data.
+* [Discovering temporal scientometric knowledge in COVID-19 scholarly production](https://paperity.org/p/281730053/discovering-temporal-scientometric-knowledge-in-covid-19-scholarly-production)
+* [YMIR](https://deepai.org/publication/ymir-a-rapid-data-centric-development-platform-for-vision-applications) - A Rapid Data-centric Development Platform for Vision Applications.
+* [DevOps in Academic Research](https://mattsegal.dev/devops-academic-research.html) - Compherensive research about DevOps including MLOps.
```

Comment:
This one recommends looking at DVC but uses Git LFS instead. Maybe not worth including?